study.md

```diff
@@ -25,7 +25,10 @@ In your own words, define what a SPA is. In your answer, be sure to cite any
 relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+SPA stands for Single Page Application. Its in a web application that serves and initail HTML template and then dynamically updates the view/UI based on user action as well as data that is passed and returned from the
+server.
```

Comment:
One important element of SPA's is that they minimize the amount of requests they send to an API. This way they are faster because after the initial GET request, they have all the data they need to manage view states and won't rely on getting more responses back from the server.